extensions/wikia/WikiaMaps/controllers/WikiaMapsMapController.class.php

```diff
@@ -11,7 +11,10 @@ class WikiaMapsMapController extends WikiaMapsBaseController {
 	 * @return Array
 	 */
 	public function getTileSets() {
-		$params = [];
+		$params = [
```

Comment:
Do we want to limit `tilesets` results always? 1) If the user creates a custom map and a list of tilesets is presented to her and 2) if the user filters presented tilesets list with a keyword?